server/runtime/src/test/resources/configuration/ServerConfigurationParserTest.xml

```diff
@@ -24,14 +24,21 @@
       </socket-bindings>
 
       <security>
+         <credential-stores>
+            <credential-store name="credentials" path="credentials.pfx">
+               <clear-text-credential clear-text="secret"/>
+            </credential-store>
+         </credential-stores>
          <security-realms>
             <security-realm name="default">
                <server-identities>
                   <ssl>
                      <keystore path="ServerConfigurationParserTest-keystore.pfx"
                                relative-to="infinispan.server.config.path"
                                keystore-password="password" alias="server" key-password="password"
-                               generate-self-signed-certificate-host="localhost"/>
+                               generate-self-signed-certificate-host="localhost">
+                        <credential-reference store="credentials" alias="keystore"/>
```

Comment:
is this credential replacing the `keystore-password` or the `key-password`?

Reply:
The keystore-password.